editor.planx.uk/src/@planx/components/ContactInput/Public.tsx

```diff
@@ -0,0 +1,156 @@
+import Card from "@planx/components/shared/Preview/Card";
+import QuestionHeader from "@planx/components/shared/Preview/QuestionHeader";
+import type { PublicProps } from "@planx/components/ui";
+import { useFormik } from "formik";
+import React from "react";
+import Input from "ui/Input";
+import InputLabel from "ui/InputLabel";
+import InputRowItem from "ui/InputRowItem";
+
+import { ERROR_MESSAGE } from "../shared/constants";
+import { getPreviouslySubmittedData, makeData } from "../shared/utils";
+import type { Contact, ContactInput } from "./model";
+import { userDataSchema } from "./model";
+
+export type Props = PublicProps<ContactInput, Contact>;
+
+interface FormProps {
+  title: string;
+  firstName: string;
+  lastName: string;
+  organisation: string;
+  phone: string;
+  email: string;
+  "name.first"?: string;
+  "name.last"?: string;
+  "company.name"?: string;
+  "phone.primary"?: string;
```

Comment:
Are these required here or leftover from trying to get them to work?

Reply:
These are indeed actually required, now fixed, thanks for catching :+1: